gpu_bdb/bdb_tools/q26_utils.py

```diff
@@ -40,3 +50,28 @@ def read_tables(config, c=None):
 
     return ss_ddf, items_ddf
 
+
+def get_clusters(client, kmeans_input_df):
+    ml_tasks = [
+        delayed(train_clustering_model)(df, N_CLUSTERS, CLUSTER_ITERATIONS, N_ITER)
+        for df in kmeans_input_df.to_delayed()
+    ]
+
+    results_dict = client.compute(*ml_tasks, sync=True)
+
+    output = kmeans_input_df.index.to_frame().reset_index(drop=True)
+    
+    if isinstance(kmeans_input_df, dask_cudf.DataFrame):
+        labels_final = dask_cudf.from_cudf(
+            results_dict["cid_labels"], npartitions=output.npartitions
+        )
+    else:
+        labels_final = dd.from_pandas(
+            pd.DataFrame(results_dict["cid_labels"]), npartitions=output.npartitions
+        )
+
+    output["label"] = labels_final.reset_index()[0]
+
+    # Based on CDH6.1 q26-result formatting
+    results_dict["cid_labels"] = output
+    return results_dict
```

Comment:
Just curious,  It seems to be that `get_clusters` is same for both 25,26 ? Should we combine it into 1 function and put it in utils ?